src/librustc_codegen_llvm/intrinsic.rs

```diff
@@ -1009,8 +1008,10 @@ fn gen_fn<'ll, 'tcx>(
         hir::Unsafety::Unsafe,
         Abi::Rust
     ));
-    let llfn = cx.define_internal_fn(name, rust_fn_sig);
```

Comment:
Is this really the only remaining instance where this function is used? Surprising.

Reply:
It was likely used in the past for shims and stuff, but a lot of that stuff is based on MIR now, which means it goes through the monomorphization collector and is uniformly declared/defined.